.idea/gradle.xml

```diff
@@ -1,26 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
```

Comment:
dont' delete this file, just remove it from this PR only.

Reply:
Noted